pkg/datasource/csvsource/bybit_tick_downloader.go

```diff
@@ -0,0 +1,101 @@
+package csvsource
+
+import (
+	"bytes"
+	"compress/gzip"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+func Download(symbol string, start time.Time) {
+	for {
+		var (
+			path     = fmt.Sprintf("pkg/datasource/csv/testdata/bybit/%s/", symbol)
```

Comment:
could we pull out the output directory as a function parameter?

Reply:
done